Automattic/wp-calypso
# Unified Pages: Group virtual home page with generic pages

#### Proposed Changes

* Render the virtual home.html page (templates) in the /pages list if the homepage displays setting is set to "latest posts" and the active theme supports FSE.
* Use the `unified-pages/virtual-home-page` flag to control the feature

![image](https://user-images.githubusercontent.com/13596067/196860034-74897543-c9ec-46f9-b8d0-a8c433b96a54.png)

https://user-images.githubusercontent.com/13596067/196390946-60b0d360-978c-449a-8003-6601a208f366.mov

(Note the final layout is the above screenshot)

TODOs

- [ ] Do we need to increase the count of the published pages by 1 if the homepage is showing the Home template?

#### Testing Instructions



* Go to "Settings > Readings", and set "Your homepage displays" to "Your latest posts"
* Go to "Pages", and you will see the virtual page, Home, listed in “Published” tabs
* Search the page by "Home", and you will see the Home page is still there
* Click on "...", and you're able to edit, view or copy the link of the Home page

#### Pre-merge Checklist






Related to https://github.com/Automattic/wp-calypso/issues/69011, https://github.com/Automattic/wp-calypso/issues/69158, https://github.com/Automattic/wp-calypso/issues/69171
